### ISSUE TITLE
Data Downloading Problem

### ISSUE THREAD

**Issue by rafaelbou** [SorourMo/95-Cloud-An-Extension-to-38-Cloud-Dataset#1]
Hi,
Thanks so much for sharing the data set. Your paper is interesting and indeed shows very intriguing results in the field.

I'm having trouble downloading the training data from the link you provided.
I can't download the entire data set, the download stops after X% of the data has been downloaded.
Have you encountered a similar problem? Do you have a way to solve the problem?

Thanks,
Rafael.

**Comment by SorourMo** [comment]
Hi Rafael,
Thank you for your interest and for letting me know. I have uploaded the dataset in Kaggle. The 95-cloud dataset is actually:  **57 Landsat 8 scenes** +  **38-Cloud dataset,**. Each of those can be downloaded via Kaggle. I have uploaded the readme file, please take a look at it here:
https://github.com/SorourMo/95-Cloud-An-Extension-to-38-Cloud-Dataset#click-here-for-downloading-57-scenes-of-95-cloud-training-set

Thanks,
Sorour

**Comment by sfensphan** [comment]
Sorour,
Thank you again for making the dataset publicly available. Forgive me as I have some confusion on how to combine the 95 dataset with the 38 dataset. I understand I will need to move the TIF into the correct training_color directory. Instead of using "training_patches_38-cloud.csv" do i delete and replace with training_patches_95-cloud.csv only? And do I need to do anything with the other csv files? Thank you

**Comment by SorourMo** [comment]
Hi @sfensphan,
No worries. You are correct about adding TIF files of the downloaded files from Kaggle to the corresponding spectral band folder in the 38-Cloud dataset (it was not possible to upload the entire 95-Cloud dataset on Kaggle, I only uploaded additional images of 95-Cloud training set).
Regarding csv files, you can use the file ```training_patches_95-cloud.csv``` for all the **training** patches and ```training_patches_95-cloud_nonempty.csv``` for all the _non-empty_ **training** patches. So you don't need deleting or editing any of the csv files.
For the testing phase, you need to use the csv file of the 38-Cloud test set (```test_patches_38-Cloud.csv```), as these two datasets have the same test set.
Hope this helps.
